system-addon/lib/ActivityStream.jsm

```diff
@@ -137,7 +137,7 @@ const PREFS_CONFIG = new Map([
   }],
   ["tippyTop.service.endpoint", {
     title: "Tippy Top service manifest url",
-    value: "https://s3.amazonaws.com/activitystream-dev-default-resources-s3bucket-1qw8m6s29v3dq/tippytop/icons.json"
+    value: "https://activity-stream-icons.services.mozilla.com/nightly/icons.json.br"
```

Comment:
If we expect "nightly" to be dynamically set based on the channel, we should use "%CHANNEL%" and have `FaviconFeed` call `Services.urlFormatter.formatURLPref("browser.newtabpage.activity-stream.tippyTop.service.endpoint")` (and get rid of the anti-pattern `new Prefs()`).

Reply:
Although for local/try/autoland builds, %CHANNEL% is "default" which doesn't seem to be a valid endpoint right now.